src/views/Playlist/PrivatePlaylist.vue

```diff
@@ -19,13 +21,31 @@ new TrackCollectionApi()
   .catch(() => {});
 </script>
 
-<template>
+<!-- <template>
   <h2>{{ trackCollection?.name }}</h2>
   <ol class="list-decimal list-inside">
     <li v-for="track in trackCollection.tracks" :key="`track-${track.id}`">
       {{ track.title }}
     </li>
   </ol>
+</template> -->
+
+<!-- This template is for test purpose. It might not work properly -->
+<template>
+  <h2>{{ trackCollection?.name }}</h2>
+  <div
+    v-for="track in trackCollection.tracks"
+    :key="track.id"
+    class="flex flex-row flex-wrap"
+  >
+    <tracksList
```

Comment:
Why do you need to dissolve the type `track` here? Wouldn't it be easier to just pass on the track instead of adding a property for each primitive type?

Reply:
Please give the naming a second chance. What is this of an element? You've called it `tracksList` -but this element doesn't look like anything I'd relate when reading it's tag - or the parameters don't fit...